cmd/generate/generate.go

```diff
@@ -189,12 +203,7 @@ func Run(o *GenerateOptions) (err error) {
 		deployment = types.SNO
 	}
 
-	isBM, err := o.utilsHelpers.IsBMInfra()
-	if err != nil {
-		return fmt.Errorf("failed to check is bm cluster %s", err)
-	}
-
-	if isBM {
+	if strings.ToLower(o.platformType) == validPlatformType[0] { // validPlatformType[0]="baremetal"
```

Comment:
no need for the toLower, and match it with the const (from the previous comment) instead of using slice[0]